lib/models/personasImport-test.js

```diff
@@ -180,31 +185,198 @@ describe('personasImport model', () => {
+        const structure = {
+          test1: {
+            columnType: COLUMN_LAST_NAME,
+          },
+          test2: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+        };
+
+        const result = validateOneFirstName(structure);
+        expect(result).to.equal(true);
+      });
+
+      it('should return false if input has multiple first name columns', () => {
+        const structure = {
+          test1: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+          test2: {
+            columnType: COLUMN_NAME,
+          },
+          test3: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+        };
+
+        const result = validateOneFirstName(structure);
+        expect(result).to.equal(false);
+      });
+    });
+
+    describe('validateOneLastName', () => {
+      it('should return true if input has 0 last name column', () => {
+        const structure = {
+          test1: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+        };
+
+        const result = validateOneLastName(structure);
+        expect(result).to.equal(true);
+      });
+
+      it('should return true if input has 1 last name column', () => {
+        const structure = {
+          test1: {
+            columnType: COLUMN_LAST_NAME,
+          },
+          test2: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+        };
+
+        const result = validateOneLastName(structure);
+        expect(result).to.equal(true);
+      });
+
+      it('should return false if input has multiple last name columns', () => {
+        const structure = {
+          test1: {
+            columnType: COLUMN_LAST_NAME,
+          },
+          test2: {
+            columnType: COLUMN_LAST_NAME,
+          },
+          test3: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+        };
+
+        const result = validateOneLastName(structure);
+        expect(result).to.equal(false);
+      });
+    });
+
+    describe('validateFullNameAndFirstOrLast', () => {
+      it('should return true if input has 0 full name column', () => {
+        const structure = {
+          test1: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+          test2: {
+            columnType: COLUMN_LAST_NAME,
+          },
+          test3: {
+            columnType: COLUMN_FIRST_NAME,
+          },
+        };
+
+        const result = validateFullNameAndFirstOrLast(structure);
+        expect(result).to.equal(true);
+      });
+
+      it('should return true if input has 0 first name and 0 last name', () => {
+        const structure = {
+          test1: {
+            columnType: COLUMN_NAME,
+          },
+          test2: {
+            columnType: COLUMN_NAME,
+          },
+        };
+
+        const result = validateFullNameAndFirstOrLast(structure);
+        expect(result).to.equal(true);
+      });
+
+      it('should return false if input has first name and full name', () => {
```

Comment:
We probably could have just defaulted to using the full name in these last two cases, but this is very explicit and I think I prefer this approach. Thanks for adding tests that were clear to understand 👍